aws/resource_aws_securityhub_account_test.go

```diff
@@ -0,0 +1,84 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/securityhub"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSSecurityHubAccount_basic(t *testing.T) {
```

Comment:
We should setup this acceptance testing similar to `TestAccAWSGuardDuty` and `TestAccAWSAppMesh`

e.g.

```go
func TestAccAWSSecurityHub(t *testing.T) {
	testCases := map[string]map[string]func(t *testing.T){
		"Account": {
			"basic":  testAccAWSSecurityHubAccount_basic,
		},
	}

	for group, m := range testCases {
		m := m
		t.Run(group, func(t *testing.T) {
			for name, tc := range m {
				tc := tc
				t.Run(name, func(t *testing.T) {
					tc(t)
				})
			}
		})
	}
}
```

```suggestion
func testAccAWSSecurityHubAccount_basic(t *testing.T) {
```

Reply:
No, its only necessary when we need grouping of tests to very explicitly limited to one-at-a-time with our TeamCity acceptance testing setup. You can still run individual testing via `-run=Thing/Subthing`

Reply:
:+1: